openslides/core/config_variables.py

```diff
@@ -130,6 +130,19 @@ def get_config_variables():
         group='General',
         subgroup='Export')
 
+    yield ConfigVariable(
+        name='general_export_pdf_fontsize',
+        default_value='10',
+        input_type='choice',
+        label='Standard font size in PDF',
+        choices=(
+            {'value': '10', 'display_name': '10'},
```

Comment:
Ok, but this is not obvious. You mean font size for standard text. Standard font size sounds like a basic fontsize from which all other font sizes are derived. Maybe change label.